pkgs/applications/networking/instant-messengers/signalbackup-tools/default.nix

```diff
@@ -1,30 +1,31 @@
-{ lib, stdenv, fetchFromGitHub, openssl, sqlite }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, openssl, sqlite }:
 
 stdenv.mkDerivation rec {
   pname = "signalbackup-tools";
-  version = "20220711";
+  version = "20220810";
 
   src = fetchFromGitHub {
     owner = "bepaald";
     repo = pname;
     rev = version;
-    sha256 = "sha256-dKU8oTQ6ECwycDN3k7NY/pKpNWH16ceJIFDnRNEA90c=";
+    sha256 = "sha256-z/RAvNUss9rNuBQvxjJQl66ZMrlxvmS9at8L/vSG0XU=";
   };
 
-  buildInputs = [ openssl sqlite ];
-  buildFlags = [
-    "-Wall"
-    "-Wextra"
-    "-Wshadow"
-    "-Wold-style-cast"
-    "-Woverloaded-virtual"
-    "-pedantic"
-    "-std=c++2a"
-    "-O3"
-    "-march=native"
+  patches = [
+    (fetchpatch {
+      name = "fix-platform-checks.patch";
+      url = "https://github.com/bepaald/signalbackup-tools/compare/20220810..a81baf25b6ba63da7d30d9a239e5b4bbc8d1ab4f.patch";
+      sha256 = "sha256-i7fuPBil8zB+V3wHHdcbmP79OZoTfG2ZpXPQ3m7X06c=";
+    })
```

Comment:
Please change this to only include [bepaald/signalbackup-tools@`47ff35e` (#73)](https://github.com/bepaald/signalbackup-tools/pull/73/commits/47ff35e85921e109c0d58d89f40fd3a7ef2d44ff) -- probably not worth including the gitignore changes 🙂

Reply:
This patch is not based on that pull request. It was initially when I submitted the pull request but I quickly changed that (see this [diff](https://github.com/NixOS/nixpkgs/compare/578b3919bf1af70ed1198586101bb8ca5df153c1..af1f959f935fb9c34a1fd1135043cbdad302205d)) when the project's author implemented the change himself in a different way (https://github.com/bepaald/signalbackup-tools/commit/a81baf25b6ba63da7d30d9a239e5b4bbc8d1ab4f). 

This patch includes a range of commits since the patch for the relevant commit doesn't cleanly apply without including a couple other trivial commits the author has made since the most recent release.